src/api/app/models/staging/staging_project.rb

```diff
@@ -133,6 +142,14 @@ def build_state
       :acceptable
     end
 
+    def relevant_checks
+      @relevant_checks ||= checks.where(status_reports: { uuid: repositories.map(&:build_id) })
+    end
+
+    def relevant_checks_for_architectures
+      @relevant_checks_for_architectures ||= checks_for_architectures.where(status_reports: { uuid: repository_architectures.map(&:build_id) })
```

Comment:
same here

Reply:
build_id is not a field, it is coming from the backend, you can not pluck it.

Reply:
It is not the first time I make this comment :see_no_evil: